Helper/Installments.php

```diff
@@ -0,0 +1,77 @@
+<?php
+/**
+ *                       ######
+ *                       ######
+ * ############    ####( ######  #####. ######  ############   ############
+ * #############  #####( ######  #####. ######  #############  #############
+ *        ######  #####( ######  #####. ######  #####  ######  #####  ######
+ * ###### ######  #####( ######  #####. ######  #####  #####   #####  ######
+ * ###### ######  #####( ######  #####. ######  #####          #####  ######
+ * #############  #############  #############  #############  #####  ######
+ *  ############   ############  #############   ############  #####  ######
+ *                                      ######
+ *                               #############
+ *                               ############
+ *
+ * Adyen Payment module (https://www.adyen.com/)
+ *
+ * Copyright (c) 2020 Adyen BV (https://www.adyen.com/)
+ * See LICENSE.txt for license details.
+ *
+ * Author: Adyen <magento@adyen.com>
+ */
+
+namespace Adyen\Payment\Helper;
+
+class Installments
+{
+    static $card_types = array(
+        'AE' => 1,
+        'ELO' => 2,
+        'HIPER' => 4,
+        'HIPERCARD' => 4,
+        'MC' => 8,
+        'VI' => 16
+    );
+
+    /**
+     *
+     * @param array $installmentsArray
+     * @return string
+     *
```

Comment:
please describe the purpose of this method as this is a very complex one.

Reply:
Added more info to the doc block to explain the purpose.